04-onion-routing.md

```diff
@@ -263,6 +264,12 @@ It is formatted according to the Type-Length-Value format defined in [BOLT #1](0
     2. data:
         * [`32*byte`:`payment_secret`]
         * [`tu64`:`total_msat`]
+    1. type: 10 (`encrypted_recipient_data`)
+    2. data:
+        * [`...*byte`:`encrypted_data`]
+    1. type: 12 (`blinding_point`)
+    2. data:
+        * [`point`:`blinding`]
 
 ### Requirements
```

Comment:
You should change the "MUST include `short_channel_id`". It may now be inside the blinded part.

Reply:
Good point. I've kind of done it only half-way currently, where I specified the route blinding algorithm but didn't really specify how to use it for payments, as I feel it may be more controversial (and isn't necessary if we want to start with onion messages only).

I'll update this part right now though, and will add more details about the payment flow later depending on how early reviews go.

Reply:
Done in https://github.com/lightningnetwork/lightning-rfc/pull/765/commits/7c02eb7594ec9ccad4c84742d5349051a6019642